lib/matplotlib/transforms.py

```diff
@@ -616,10 +616,22 @@ def expanded(self, sw, sh):
         a = np.array([[-deltaw, -deltah], [deltaw, deltah]])
         return Bbox(self._points + a)
 
-    def padded(self, p):
-        """Construct a `Bbox` by padding this one on all four sides by *p*."""
+    def padded(self, pw, ph=None):
```

Comment:
After just expiring a number of deprecations which involved name changes of non-optional arguments, I would suggest adding a `@_api.rename_parameter("3.8", "p", "pw")`.

In addition, may one should go all in and spell out the parameter name?

Reply:
I picked very short names for consistency with (some of) the other methods on this class, but maybe `w_pad` and `h_pad` would be better?  That would make them consistent with those for the relatively recently added constrained layout engine.

Thanks for the pointer on `_api.rename_parameter`.  I tend to forget that the difference between positional and keyword args in python is only pretend!